topics/admin/tutorials/ansible-galaxy/group_vars/galaxyservers.yml

```diff
@@ -18,20 +18,23 @@ galaxy_manage_paths: true
 galaxy_layout: root-dir
 galaxy_root: /srv/galaxy
 galaxy_user: {name: galaxy, shell: /bin/bash}
-galaxy_commit_id: release_20.05
+galaxy_commit_id: release_20.09
 galaxy_config_style: yaml
 galaxy_force_checkout: true
+
 miniconda_prefix: "{{ galaxy_tool_dependency_dir }}/_conda"
 miniconda_version: "4.6.14"
+miniconda_manage_dependencies: false
```

Comment:
@hexylena That's not mentioned in the tutorial AFAICS, which one is right?

Reply:
that should be there yea. it should be mentioned. this commit got forgotten and added, its been done in several beanches. will fix in jamuary

Reply:
thnaks for the PR to fix this!